documentation/release-notes/topics/version_4.rst

```diff
@@ -0,0 +1,103 @@
+- Previously API client permissions such as Role and Access Policies we not editable. With this version, API user permissions can be modified by an Administrator
+
+===========================================
+Enhanced Feature - Power Path Visualization
+===========================================
+- Power path visualization will allow you to double-click and explore various nodes in the power path
+- Power path can now be exported to PDF and various image formats
+
+=========================================
+Enhanced Feature - Credentials Management
+=========================================
+- Credentials management will not allow you to view multiple passwords at once
+- Credentials management has been enhanced to create an application log when an Administrator views the password within a credential record
+- Credentials management API has been enhanced to not allow an Administrator to view multiple passwords within a credentials collection
+
+============================================
+Enhanced Feature - New Troubleshooting Tools
+============================================
+- Net-SNMP docker container
+- SNMP Get troubleshooting tool
+- The BacnetIpWalkerCli diagnostic tool has been improved to allow binding to different ports
+
+======================================
+Other notable changes and improvements
+======================================
+- Any API route that has been deprecated before this release has been removed
+- The Hyperview API link under the Help navigation menu has been renamed to "API Explorer"
+- Within the connectivity feature, Connection Type has been renamed to Media Type
+- The Rack PDU layout can now be exported to MS Excel
+- The document storage calculation will take into account user inbox message space usage
+- The License page has been improved to show Licensed and consumed connections
+- Various improvements to the sensor card visualization
+- Various improvements to the discovery subsystem that should improve speed
+- Various additional improvements to manual discovery and discovery abort controls in the Linux version of the data collector
+
+============
+Known issues
+============
+- **AS-13409** When there is a pending edit, the browser doesn't display the unsaved changes alert when closing a tab, reloading, or navigating to a new URL
+
+=================
+Notable bug fixes
+=================
+- **AS-12012** Fixed a bug that caused a discovery CIDR range details to not display when adding an address range to a discovery
+- **AS-13088** Fixed a bug that caused shaped edits to not work after a floor plan layout shape is saved
+- **AS-13157** Fixed a bug that caused certain component-level sensors not to trigger thresholds
+- **AS-13257** Fixed a bug that caused the 3D layout popover information to not be localized
+- **AS-13360** Fixed a bug in the asset discovery report, where under certain conditions the "Credential Description" remained empty if the asset fails to discover with SNMPv3
```

Comment:
There were two levels with this bug.  It was always a problem with failed v3 and a problem for success or failure with blank passwords for any protocol. 

We did not know about the second level until reg testing.  Not sure if you want to tweak your verbiage.

Reply:
I think the verbiage is good for now. Thanks!